PBS_gadi_runPipeline.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+  
+#PBS -P er01 
+#PBS -N Somatic-shortV
+#PBS -l walltime=02:00:00
+#PBS -l ncpus=1
+#PBS -l mem=60GB
+#PBS -W umask=022
+#PBS -q copyq
+#PBS -e Somatic-shortV-nf.e
+#PBS -o Somatic-shortV-nf.o
+#PBS -l wd
+#PBS -l storage=scratch/er01+gdata/er01
+#PBS -l jobfs=10GB
+
+module load nextflow/22.04.3
+module load singularity
+
+
+export NXF_SINGULARITY_CACHEDIR=/scratch/$PROJECT/$(whoami)/singularity
+
+# Fill in these variables for your run
+
+samples=/scratch/er01/ndes8648/pipeline_work/nextflow/INFRA-83-Somatic-ShortV/Somatic-shortV-nf/samples.csv
+ponvcf=/scratch/er01/ndes8648/pipeline_work/nextflow/INFRA-83-Somatic-ShortV/Somatic-shortV-nf/pon.vcf.gz
+ref=/g/data/er01/SIH-HPC-WGS/Reference/hs38DH.fasta
+small_exac_common=/g/data/er01/SIH-HPC-WGS/Reference/gatk-best-practices/somatic-hg38/small_exac_common_3.hg38.vcf.gz
+
+intervalList_path=/g/data/er01/SIH-HPC-WGS/Reference/BQSR_apply_intervals_13
+
+outDir=results
+whoami=npd561  
```

Comment:
This does not need to be defined, `$(whoami)` is a bash command that picks up username